src/components/layer-controller/styles.js

```diff
@@ -22,19 +24,32 @@ export const useOptionStyles = makeStyles(theme => ({
   },
 }));
 
-export const useExpansionPanelStyles = makeStyles(() => ({
+const sharedControllerStyles = {
+  width: '100%',
+  flexDirection: 'column',
+};
+
+export const useControllerSectionStyles = makeStyles(() => ({
   root: {
-    width: '100%',
-    flexDirection: 'column',
+    ...sharedControllerStyles,
+    padding: '0px 8px',
   },
 }));
 
-export const useExpansionPanelSummaryStyles = makeStyles(theme => ({
+export const StyledExpansionPanelDetails = withStyles(() => ({
+  root: {
+    ...sharedControllerStyles,
+    padding: '8px 8px 24px 8px',
+  },
+}))(ExpansionPanelDetails);
+
+export const StyledExpansionPanelSummary = withStyles(theme => ({
   root: {
     top: theme.spacing(-1),
```

Comment:
Do you notice anything when removing this?  When collapsed it looks a little nicer.
```suggestion
```
With this suggested change:
<img width="241" alt="Screen Shot 2021-05-28 at 8 30 33 PM" src="https://user-images.githubusercontent.com/43999641/120052493-9902fc80-bff3-11eb-842b-5523f7216c2e.png">
Without:
<img width="270" alt="Screen Shot 2021-05-28 at 8 32 00 PM" src="https://user-images.githubusercontent.com/43999641/120052531-c5b71400-bff3-11eb-8631-524bdea8e86f.png">
Previous:
<img width="246" alt="Screen Shot 2021-05-28 at 8 32 25 PM" src="https://user-images.githubusercontent.com/43999641/120052548-d4053000-bff3-11eb-83e4-113fb74df997.png">
Not what it was but a bit of an improvement.  Thanks again for this!